Generator/OdmGenerator.php

```diff
@@ -72,7 +72,14 @@ protected function generateMappingDocumentFiles(OutputInterface $output, BundleM
                 $output->writeln(sprintf('   ~ <info>%s</info>', $fileName));
             } else {
                 $output->writeln(sprintf('   + <info>%s</info>', $fileName));
-                copy($src_file, $dest_file);
+
+                $mappingEntityTemplate = file_get_contents($src_file);
+
+                $string = Mustache::replace($mappingEntityTemplate, array(
```

Comment:
I'd say use a bundle and inject [this service](https://github.com/bobthecow/BobthecowMustacheBundle/blob/master/Resources/config/mustache.xml#L16). Also, why use Mustache over Twig ?